src/target/llvm/codegen_llvm.cc

```diff
@@ -1871,7 +1871,11 @@ llvm::Value* CodeGenLLVM::VisitExpr_(const BroadcastNode* op) {
   value = builder_->CreateInsertElement(undef, value, zero);
 #if TVM_LLVM_VERSION >= 110
   llvm::ElementCount ec =
+#if TVM_LLVM_VERSION >= 160
       llvm::ElementCount::get(dtype.get_lanes_or_vscale_factor(), dtype.is_scalable_vector());
+#else
+      llvm::ElementCount(dtype.get_lanes_or_vscale_factor(), dtype.is_scalable_vector());
+#endif
```

Comment:
Curious about this change, I tried building with llvm-13 locally but it failed. Also it seems the build for ci-cpu (which uses llvm-15) also failed: https://ci.tlcpack.ai/blue/organizations/jenkins/tvm-cpu/detail/PR-16808/2/pipeline. What version of llvm caused the failure?

Reply:
@lhutton1 

* Updated now.
```
+#if TVM_LLVM_VERSION >= 120
       llvm::ElementCount::get(dtype.get_lanes_or_vscale_factor(), dtype.is_scalable_vector());
+#else
+      llvm::ElementCount(dtype.get_lanes_or_vscale_factor(), dtype.is_scalable_vector());
+#endif
```

I double checked compilation with llvm={10,11,12,13,14,15,16,17,18} .

Reply:
Thanks, that works now :)

Reply:
@lhutton1 

> Thanks, that works now :)

Also thanks for the catch !

Sorry for the confusion my double checks took **some longer time**
The threshold of  ```getter``` turned to be at ```>= 120``` after exhaustive checks.